include/deal.II/matrix_free/matrix_free.h

```diff
@@ -2689,11 +2689,12 @@ MatrixFree<dim, Number, VectorizedArrayType>::n_active_fe_indices() const
 template <int dim, typename Number, typename VectorizedArrayType>
 unsigned int
 MatrixFree<dim, Number, VectorizedArrayType>::get_cell_active_fe_index(
-  const std::pair<unsigned int, unsigned int> range) const
+  const std::pair<unsigned int, unsigned int> range, unsigned int dof_handler_index) const
```

Comment:
I don't think we need to pass here `dof_handler_index`. I think it is enough to use the first one.

Reply:
I was actually not sure about it. I changed it now.

Reply:
I think there could be made a case for this (at least as a general interface): We could have multiple `DoFHandler` objects, and it could be that the second or third `DoFHandler` in the list of `MatrixFree::dof_handlers` could be the one with `hp` capability, and then we would miss this. Currently, we let `dof_handlers[0]` determine this information, see e.g. https://github.com/dealii/dealii/blob/578fcb0c15df4b88c42f71d676ac29a93ae007f3/include/deal.II/matrix_free/matrix_free.templates.h#L798-L812 and https://github.com/dealii/dealii/blob/578fcb0c15df4b88c42f71d676ac29a93ae007f3/include/deal.II/matrix_free/matrix_free.templates.h#L1557-L1564 and many other places assuming the index from `dh[0]`, but I think that could be relaxed and then the interface makes sense. That said, we can opt for a small change now and check it later.